cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -1618,7 +1618,8 @@ data:
                 "hideUnclaimedVolumes": "{{ default false ((.Values.kubecostProductConfigs).hideUnclaimedVolumes) }}",
                 "hideCloudIntegrationsUI": "{{ default false ((.Values.kubecostProductConfigs).hideCloudIntegrationsUI) }}",
                 "hideBellIcon": "{{ default true ((.Values.kubecostProductConfigs).hideBellIcon) }}",
-                "hideTeams": "{{ default false ((.Values.kubecostProductConfigs).hideTeams) }}"
+                "hideTeams": "{{ default false ((.Values.kubecostProductConfigs).hideTeams) }}",
+                "clickhouseEnabled": "{{ .Values.kubecostAggregator.useDBv3 }}"
```

Comment:
```suggestion
                "DBv3Enabled": "{{ .Values.kubecostAggregator.useDBv3 }}"
```

Reply:
Agree with this suggestion! It's easiest to reason about these fields, when they closely match their Helm configuration name.